tidb/src/main/java/io/tidb/bigdata/tidb/ClientConfig.java

```diff
@@ -25,7 +25,7 @@ public final class ClientConfig {
 
   public static final String TIDB_DRIVER_NAME = "io.tidb.bigdata.jdbc.TiDBDriver";
```

Comment:
It's not used. Maybe we should delete it.

Reply:
@humengyu2012 Thanks for your review. I have removed it.